_posts/2024-03-15-customer-experience-outcomes.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title:  "Customer experience outcomes of focused, user-centered product design improvements"
+authors:
+  - apasun
+date:   2024-03-15 0:00:10 -0700
+categories:
+  - community
+excerpt: The OpenSearch Project invites the OpenSearch community to explore the future of search, analytics, and generative AI at the first OpenSearch user conference in Europe. Join us in Berlin on May 6 & 7 and learn how to build powerful applications and get the most out of your OpenSearch deployments.
+meta_keywords: 
+meta_description: 
+---
+
+## User experience challenges
+
+A large proportion of software improvements focus on features and functionality. It is common practice for product managers and developers to incrementally add engineering capabilities. When the implementation of a seamless experience is not taken into account, the result can be a fragmented user experience. While these changes may not affect the perceived experience of a high-code user, low-code users report excess frustration and a lack of satisfaction with an experience when changes to software do not aid in accomplishing set outcomes that a software experience intends to deliver.
+
+When navigating software, users utilize two types of memory: episodic and semantic. Episodic memory is grounded in the sequence of time, while semantic memory is conceptual. We, as humans, are capable of experiencing our online environment in both ways and often switch between the two types of memory as we navigate software. When functionality is enhanced incrementally, however, navigation that fails to account for the logical buildup of user tasks in a software workflow can quickly degrade the experience. Such product experiences end up ill designed and requiring ongoing customer support to help coach, train, and assist the user.
+
+## Is there a way to measure user experience?
+The [Customer Outcome Framework](https://productmanagementuniversity.com/launches-customer-outcome-framework/) employs a top-down view of the customer experience. Customer Experience Outcomes (CXOs) [define critical tasks that a product must perform and how well those tasks must be performed](https://lithespeed.com/drive-input-customer-experience-outcomes/#:~:text=What%20is%20a%20Customer%20Experience%20Outcome%20or,a%20plan%20for%20success:%20%E2%80%9CIf%20we%20do). A comprehensive set of CXO measures may include [quality and loyalty measures](https://www.forrester.com/research/cx-index/). Additional outcome measures may include [satisfaction and frustration scores](https://www.dynatrace.com/news/blog/user-experience-score-the-one-metric-to-rule-them-all/), [UMUX](https://testscience.org/measuring-usability/#UMUX)[measures](https://testscience.org/measuring-usability/#UMUX), [UMUX-lite measures](https://testscience.org/measuring-usability/#UMUX-LITE), or [SUS scores](https://testscience.org/measuring-usability/#SUS).
```

Comment:
Should "Customer Experience Outcomes (CXOs)" be linked instead of the text starting with "define..."?